Libraries/LibWeb/XPath/XPathResult.h

```diff
@@ -19,16 +19,16 @@ class XPathResult : public Bindings::PlatformObject {
     GC_DECLARE_ALLOCATOR(XPathResult);
 
 public:
-    unsigned short const ANY_TYPE = 0;
-    unsigned short const NUMBER_TYPE = 1;
-    unsigned short const STRING_TYPE = 2;
-    unsigned short const BOOLEAN_TYPE = 3;
-    unsigned short const UNORDERED_NODE_ITERATOR_TYPE = 4;
-    unsigned short const ORDERED_NODE_ITERATOR_TYPE = 5;
-    unsigned short const UNORDERED_NODE_SNAPSHOT_TYPE = 6;
-    unsigned short const ORDERED_NODE_SNAPSHOT_TYPE = 7;
-    unsigned short const ANY_UNORDERED_NODE_TYPE = 8;
-    unsigned short const FIRST_ORDERED_NODE_TYPE = 9;
+    static unsigned short const ANY_TYPE = 0;
+    static unsigned short const NUMBER_TYPE = 1;
+    static unsigned short const STRING_TYPE = 2;
+    static unsigned short const BOOLEAN_TYPE = 3;
+    static unsigned short const UNORDERED_NODE_ITERATOR_TYPE = 4;
+    static unsigned short const ORDERED_NODE_ITERATOR_TYPE = 5;
+    static unsigned short const UNORDERED_NODE_SNAPSHOT_TYPE = 6;
+    static unsigned short const ORDERED_NODE_SNAPSHOT_TYPE = 7;
+    static unsigned short const ANY_UNORDERED_NODE_TYPE = 8;
+    static unsigned short const FIRST_ORDERED_NODE_TYPE = 9;
```

Comment:
Please add these `static`s in the previous commit, where you introduced these lines.